src/open_inwoner/templates/pages/contactmoment/list.html

```diff
@@ -1,10 +1,33 @@
 {% extends 'master.html' %}
-{% load i18n anchor_menu_tags card_tags grid_tags icon_tags link_tags list_tags pagination_tags utils %}
+{% load i18n anchor_menu_tags button_tags card_tags form_tags grid_tags icon_tags link_tags list_tags pagination_tags utils %}
 
 {% block content %}
     <h1 class="utrecht-heading-1" id="contactmomenten">{{ page_title }} ({{ contactmomenten|length }})</h1>
 
+    {# title + scroll down button #}
+    <section class="contactmomenten__scrolldown">
+        <div class="form-container">
+            <div class="form-heading">
+                <h2 class="h2">{% trans "Stel een vraag" %}</h2>
+                <p class="p">
+                {% blocktrans trimmed %}
+                    Heeft u een vraag? Dan kunt u deze hier stellen.
+                    Een van onze medewerkers beantwoord uw vraag z.s.m.
+                {% endblocktrans %}
+                </p>
+            </div>
+            {% button href="#question-form" text=_("Stel een vraag") title=_("Stel een vraag") primary=True icon="arrow_downward" icon_position="after" %}
```

Comment:
The design isn't done yet but I would still like to see this wrapped with a Div with class `button-row` to give it correct padding,
see also: '_src/open_inwoner/components/templates/components/Button/ButtonRow.html_'